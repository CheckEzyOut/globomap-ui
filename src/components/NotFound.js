import React from 'react';
import { Link } from 'react-router-dom';
import './css/NotFound.css';


class NotFound extends React.Component {
  render() {
    return (
      <div className="not-found">
        <div className="not-found-title">
          <h1>
            <span className="status">404</span><br />
            <strong className="message">Not found!</strong>
          </h1>
          <p>
            <Link to="/" className="btn-back">
              &laquo;&nbsp;Go back
            </Link>
          </p>
        </div>
      </div>
    );
  }
}

export default NotFound;
