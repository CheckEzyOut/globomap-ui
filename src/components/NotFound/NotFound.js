/*
Copyright 2018 Globo.com

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
*/

import React from 'react';
import { Link } from 'react-router-dom';
import './NotFound.css';


class NotFound extends React.Component {
  render() {
    return (
      <div className="not-found">
        <div className="not-found-title">
          <h1>
            <span className="status">404</span><br />
            <strong className="message">Not found!</strong>
          </h1>
          <p>
            <Link to="/" className="btn-back">
              &laquo;&nbsp;Go back
            </Link>
          </p>
        </div>
      </div>
    );
  }
}

export default NotFound;
