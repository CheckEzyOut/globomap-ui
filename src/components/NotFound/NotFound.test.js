import React from 'react';
import { shallow } from 'enzyme';
import { NotFound } from './NotFound';

describe("NotFound", function() {

  it('should render NotFound component', () => {
    const component = shallow(<NotFound />);
    expect(component).toMatchSnapshot();
  });

});
