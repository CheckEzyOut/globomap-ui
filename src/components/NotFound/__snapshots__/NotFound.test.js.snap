// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should render NotFound component 1`] = `
<div
  className="not-found"
>
  <div
    className="not-found-title"
  >
    <h1>
      <span
        className="status"
      >
        404
      </span>
      <br />
      <strong
        className="message"
      >
        Not found!
      </strong>
    </h1>
    <p>
      <Link
        className="btn-back"
        replace={false}
        to="/"
      >
        « Go back
      </Link>
    </p>
  </div>
</div>
`;
